test/unit/spec/nbextensions/bulkImportCell/tabs/configure-spec.js

```diff
@@ -125,44 +137,74 @@ define([
                 label: 'view',
             },
         ].forEach((testCase) => {
-            it(`should start in ${testCase.label} mode`, () => {
-                const model = Props.make({
-                    data: Object.assign({}, TestBulkImportObject, { state: initialState }),
-                    onUpdate: () => {
-                        /* intentionally left blank */
-                    },
+            describe(`${testCase.label} mode`, () => {
```

Comment:
A bit of refactoring. Test on each mode startup: it starts, the file type icons are correct, and parameter evaluation happens (or doesn't) as expected.